thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/util/AlertFilterHelper.java

```diff
@@ -2,31 +2,24 @@
 
 import com.linkedin.thirdeye.datalayer.dto.AnomalyFunctionDTO;
 import com.linkedin.thirdeye.datalayer.dto.MergedAnomalyResultDTO;
-import com.linkedin.thirdeye.datalayer.pojo.AnomalyFunctionBean;
 import com.linkedin.thirdeye.detector.email.filter.AlertFilter;
-import com.linkedin.thirdeye.detector.email.filter.AlphaBetaAlertFilter;
-import com.linkedin.thirdeye.detector.email.filter.AlertFilterType;
-import com.linkedin.thirdeye.detector.email.filter.DummyAlertFilter;
+import com.linkedin.thirdeye.detector.email.filter.AlertFilterFactory;
+
 import java.util.ArrayList;
-import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+
+
 public class AlertFilterHelper {
-  public static final String FILTER_TYPE_KEY = "type";
   private static final Logger LOG = LoggerFactory.getLogger(AlertFilterHelper.class);
+  private AlertFilterFactory alertFilterFactory;
 
-  public static AlertFilter getAlertFilter(AlertFilterType filterType) {
-    switch (filterType) {
-      case ALPHA_BETA:
-        return new AlphaBetaAlertFilter();
-      case DUMMY:
-      default:
-        return new DummyAlertFilter();
-    }
+  public AlertFilterHelper(String alertFilterConfigPath){
+    this.alertFilterFactory = new AlertFilterFactory(alertFilterConfigPath);
```

Comment:
If applyFiltrationRule takes as input an AlertFilterFactory, then AlertFilterHelper doesn't need to keep a factory. You could remove the variable: alertFilterFactory of this class.